chef/data_bags/crowbar/migrate/provisioner/013_upgrade_autoyast_repos.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  unless a['suse'].empty? && a['suse']['autoyast'].empty? && a['suse']['autoyast']['repos'].empty?
```

Comment:
Since we're coming post-release, we might need to check that there's no `common`, `suse-11.3` or `suse-12.0` as attributes inside `a['suse']['autoyast']['repos']`.

Something like:

```
if %w(common suse-11.3 suse-12.0).select {|k|
  a['suse']['autoyast']['repos'].keys.include? k
}.empty?
```
(totally untested)